app/assets/stylesheets/modules/mfa.css

```diff
@@ -13,3 +13,24 @@
     gap: 1rem;
   }
 }
+
+.mfa__header {
+  display: inline-block;
+  padding-top: 0px !important;
+}
+
+.pill {
+  margin-bottom: 20px;
+  margin-left: 10px;
+  width: 10%;
+  max-width: 320px;
+  background-color: #e9573f;
+  color: #ffffff;
+  transition-duration: 0.25s;
+  transition-property: opacity;
```

Comment:
I don't think we're animating or transitioning anything here. I don't think these rules actually do anything 🤔 .